packages/lesswrong/lib/collections/posts/views.js

```diff
@@ -546,13 +546,11 @@ Posts.addView("communityResourcePosts", function () {
 // No index needed
 
 Posts.addView("sunshineNewPosts", function () {
-  const twoDaysAgo = moment().subtract(2, 'days').toDate();
   return {
     selector: {
       reviewedByUserId: {$exists: false},
       frontpageDate: {$in: [false,null] },
-      createdAt: {$gt: twoDaysAgo},
-      authorIsUnreviewed: null
+      authorIsUnreviewed: true
```

Comment:
What is this? This looks incorrect.
```suggestion
      authorIsUnreviewed: null
```